README.md

```diff
@@ -17,12 +17,12 @@ the square brackets (for multiple-choice questions).
 
 Select all that apply:
 ```
-[] Strings
-[] Booleans
+[X] Strings
+[X] Booleans
 [] Undefined
 [] NaN
-[] Integers
-[] Arrays
+[X] Integers
+[X] Arrays
```

Comment:
In JavaScript, an array is a special type of object. Therefore typeof [1,2,3,4] returns object.  Therefore, they aren't officially a data type.